frontend/src/assets/icons/BxsMagicWand.tsx

```diff
@@ -0,0 +1,20 @@
+export const BxsMagicWand = (
```

Comment:
@KenLSM To check if we can avoid manual imports of it (licensing problems)

Reply:
Switched to to use react-icons -- had to also update package to its latest version.

Seems compatible! (left new, right previous)
<img width="902" alt="Screenshot 2024-10-30 at 6 28 27 PM" src="https://github.com/user-attachments/assets/633e4bc6-f505-4698-8915-77fe6ac5eb92">